emain/emain-util.ts

```
@@ -170,6 +170,9 @@
 export function waveKeyToElectronKey(waveKey: string): string {
     const waveParts = waveKey.split(":");
     const electronParts: Array<string> = waveParts.map((part: string) => {
+        const digitRegexpMatch = new RegExp("^c\{Digit(\d)\}$").exec(part);
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)